Components/Alert/alert.css

```diff
@@ -0,0 +1,117 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    --simple-alert-bg-color: #bae6fd;
+    --success-alert-bg-color: #d1e7dd;
+    --error-alert-bg-color: #f8d7da;
+    --alert-color: black;
+    --alert-icon-color: black;
+    /* focus color is seen when you use tab key to access buttons or links */
+    --alert-button-focus-outline-color: black;
```

Comment:
try to maintain consistency if using hex plz stick to it.

Reply:
Ok sure, thanks